<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="tacticScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>551</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>551</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>135</width>
       <height>558</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="tactic1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>
Tactic 1:
Ambush
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tactic2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>
Tactic 2:
Encirclement
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tactic3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>
Tactic 3:
Defeat-In-Detail
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tactic4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>
Tactic 4:
Elastic Defense
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tactic5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>
Tactic 5:
Infiltration
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tactic6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>
Tactic 6:
Feigned Retreat
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="freeplay">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>
Free Play

</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="infoFrame">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>339</y>
      <width>651</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="categoryContainer" native="true">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget { background: lightgrey }</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="descriptionButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="historyButton">
          <property name="text">
           <string>History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="funFactsButton">
          <property name="text">
           <string>Fun Facts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quizButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Quiz</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="textBoxContainer">
       <property name="styleSheet">
        <string notr="true">QWidget { background: white }</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="textLayout">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>647</width>
          <height>151</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="textBox">
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>759</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>661</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="colorPalette4Frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>343</y>
      <width>801</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="freePlayBackground" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>651</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgray;</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="playableSoldierOptions">
      <item>
       <widget class="QPushButton" name="placeAllyButton">
        <property name="text">
         <string>Place Ally</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="placeEnemyButton">
        <property name="text">
         <string>Place Enemy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="freeSimPlayButton">
        <property name="text">
         <string>Move Soldiers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetAnimationButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Reset Simulation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="freePlayPartitionLine">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>21</width>
       <height>190</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="staticSoldierLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Soldier:</string>
     </property>
    </widget>
    <widget class="QLabel" name="soldierName">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setTrajectoryPoint">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Trajectory Point</string>
     </property>
    </widget>
    <widget class="QPushButton" name="endFreePlayButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>100</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>End Freeplay</string>
     </property>
    </widget>
    <widget class="QLabel" name="staticPoint">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Point Selected:</string>
     </property>
    </widget>
    <widget class="QLabel" name="coordinates">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>100</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>---</string>
     </property>
    </widget>
   </widget>
   <zorder>colorPalette4Frame</zorder>
   <zorder>backgroundLabel</zorder>
   <zorder>tacticScrollArea</zorder>
   <zorder>infoFrame</zorder>
   <zorder>playButton</zorder>
   <zorder>freePlayBackground</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
